### PR TITLE
MJ tweak support workers for sepa emails

### PR DESCRIPTION
**Pull request by michaelbjacobson**
### What does this PR do?
This PR, which should be merged alongside changes to relevant Braze campaign, tweaks the `support-workers` repo to further make it compatible with SEPA direct debit recurring contributions.

### CODE REVIEW

File: support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala [hunk 1]
```diff
@@ -9,9 +9,9 @@ import org.joda.time.DateTime
 import scala.concurrent.{ExecutionContext, Future}
 
 class ContributionEmailFields(
-  getMandate: String => Future[Option[String]],
-  created: DateTime,
-) {
+                               getMandate: String => Future[Option[String]],
+                               created: DateTime,
+                             ) {
```
Comment (TomPretty) [inline support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala:14]: I guess we don't want this?

Comment (michaelbjacobson) [reply support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala:14]: Good shout. I just left this alone, but yeah, it's only being used in unit tests now. @tomrf1 are you happy for this to be removed?

Comment (TomPretty) [reply support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala:14]: Oh I just meant the formatting!

Comment (TomPretty) [reply support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala:14]: would definitely advocate for removing unused code though!

Comment (michaelbjacobson) [reply support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala:14]: I was just about to comment again saying that I missed another use of the type! I'm not firing on all cylinders this week! Yeah, I'll fix the inconsistent formatting.

Comment (tomrf1) [reply support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala]: looks like there's still an extra space there!

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**